aptos-move/framework/cached-packages/src/lib.rs

```diff
@@ -2,20 +2,18 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use include_dir::{include_dir, Dir, DirEntry};
-use move_deps::{
-    move_binary_format::file_format::CompiledModule, move_bytecode_utils::Modules,
-    move_core_types::abi::ScriptABI,
-};
+use move_deps::{move_binary_format::file_format::CompiledModule, move_core_types::abi::ScriptABI};
 use once_cell::sync::Lazy;
+use std::collections::HashSet;
 
 pub mod aptos_stdlib;
 pub mod aptos_token_stdlib;
 mod generated_aptos_txn_builder;
 mod generated_token_txn_builder;
 
-static PACKAGE: Dir<'static> = include_dir!("$OUT_DIR/framework");
+static PACKAGE: Dir<'static> = include_dir!("$OUT_DIR");
 
-static MODULE_BLOBS: Lazy<Vec<Vec<u8>>> = Lazy::new(|| load_modules("build"));
+static MODULE_BLOBS: Lazy<Vec<Vec<u8>>> = Lazy::new(load_modules);
```

Comment:
Did these paths change upstream?  Would this break compatibility with anything that's processing it?

Reply:
I validate all the callsites using this cached-package and updated them. all tests passed now.